src/NServiceBus.Core/Settings/SettingsHolder.cs

```diff
@@ -130,6 +143,26 @@ public T GetOrDefault<T>(string key)
         }
 
         /// <summary>
+        /// Gets the default value for the setting or <code>default(T).</code>.
```

Comment:
Can we be more specific here? This method returns value from Defaults collection or default value of T. It's not obvious from description.